app/server/lib/SamlConfig.ts

```diff
@@ -32,6 +32,15 @@
  *        Comma-separated list of paths for certificates from identity provider, PEM format.
  *    env GRIST_SAML_IDP_UNENCRYPTED
  *        If set and non-empty, allow unencrypted assertions, relying on https for privacy.
+ *    env GRIST_SAML_ATTR_FIRSTNAME
+ *        If set and non-empty, contains the SAML attribute for the user first name.
```

Comment:
I would think it could be more clear to phrase like below:
```
If set and non-empty, determines the user's name attribute from the IdP Response
```

What do you think?